sys/nginx/nginx.conf

```diff
@@ -11,6 +13,29 @@ server {
 
     root /application/current/public;
 
+    set $skip_cache 0;
+    if ($request_method = POST) { set $skip_cache 1; }
+    if ($query_string != "") { set $skip_cache 1; }
+    if ($request_uri ~ "^/$") { set $skip_cache 1; }
+
+    fastcgi_cache cached_badges;
+    fastcgi_cache_bypass $skip_cache;
+    fastcgi_cache_background_update on;
+    fastcgi_cache_key "$scheme$request_method$host$request_uri";
+    fastcgi_cache_use_stale error timeout invalid_header http_500 http_503;
+    fastcgi_cache_valid 200 1m;
+    fastcgi_no_cache $skip_cache;
+    add_header X-FastCGI-Cache $upstream_cache_status;
+
+    gzip on;
+    gzip_vary on;
+    gzip_proxied expired no-cache no-store private auth;
+    gzip_types text/plain text/css text/xml text/javascript application/javascript application/x-javascript application/xml;
+
+    #location ~* \.(jpg|jpeg|png|gif|ico|css|js)$ {
```

Comment:
I believe it is missing the php upstream and it tries to serve a static asset that doesn't exist.

```
2020/05/22 16:21:43 [error] 18868#0: *42501 open() "/application/current/public/jetbrains/phpstorm-workshop/v/stable.png" failed (2: No such file or directory), client: 47.155.246.88, server: poser.pugx.org, request: "GET /jetbrains/phpstorm-workshop/v/stable.png HTTP/1.1", host: "poser.pugx.org"
```

Reply:
I left out svgs from the regex as I thought we were serving badges only as svgs (so svgs didn't get cached).

I think the problem here is because of `config/routes/badge_redirect.yaml` as we redirect requests to `.png` to the controller that generates them (didn't know about it, sorry!)

I'm ok with rolling this back for now, maybe we can fine tune this regex

Reply:
maybe we could only enable `ico|css|js`?

Reply:
done